projects/demo/src/modules/app/landing/index.html

```diff
@@ -1,7 +1,6 @@
 <img
     alt="taiga"
     src="assets/images/taiga.svg"
-    tuiLoading="lazy"
```

Comment:
shouldn't we keep the loading="lazy" attribute?

Reply:
fixed